Resources/Prototypes/Entities/Clothing/Back/smuggler.yml

```diff
@@ -0,0 +1,1009 @@
+        prob: 0.02
+        maxAmount: 2
+        orGroup: GizmosPayload
+      - id: ExplosivePayload
+        prob: 0.02
+        maxAmount: 2
+        orGroup: GizmosPayload
+      - id: SpaceCash
+        prob: 0.02
+        orGroup: GizmosPayload #2x small
+
+      - id: SpaceCash100
+        prob: 0.98
+        orGroup: Circuitboards
+      - id: ChemDispenserMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: SyndicateMicrowaveMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: HydroponicsTrayMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: DawInstrumentMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: PortableGeneratorPacmanMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: PortableGeneratorSuperPacmanMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: HellfireFreezerMachineCircuitBoard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: HellfireHeaterMachineCircuitBoard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: ReagentGrinderMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: ReagentGrinderIndustrialMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: BoozeDispenserMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: MiniGravityGeneratorCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: AmmoTechFabCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: CryoPodMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: PowerCageRechargerCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: ShuttleConsoleCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: TurboItemRechargerCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: SpaceCash
+        prob: 0.05
+        orGroup: Circuitboards #1x small
+
+- type: entity
+  parent: ClothingBackpackSatchelSmuggler
+  id: ClothingBackpackSatchelSmugglerThree
+  noSpawn: true
+  components:
+  - type: Sprite
+    sprite: Clothing/Back/Satchels/smuggler.rsi
+    state: icon
+  - type: StorageFill
+    contents:
+
+      - id: SpaceCash100
+        prob: 0.90
+        orGroup: PresentsOrToys
+      - id: PresentRandom
+        prob: 0.05
+        maxAmount: 3
+        orGroup: PresentsOrToys
+      - id: ToyFigurineHamlet
+        prob: 0.01
+        orGroup: PresentsOrToys
+      - id: ToyFigurineSpaceDragon
+        prob: 0.01
+        orGroup: PresentsOrToys
+      - id: ToyFigurineQueen
+        prob: 0.01
+        orGroup: PresentsOrToys
+      - id: ToyFigurineRatKing
+        prob: 0.01
+        orGroup: PresentsOrToys
+      - id: PresentRandomUnsafe
+        prob: 0.0015
+        orGroup: PresentsOrToys
```

Comment:
This has the potential to give admin items and other things players probably shouldn't have access to regularly.